# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../runit/patches/ntp-etc_ntp.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./sntp/header.h.orig	2006-12-04 21:59:05.000000000 +0200
+++ ./sntp/header.h	2006-12-04 21:59:39.000000000 +0200
@@ -19,10 +19,10 @@
 #define MAX_SOCKETS        10          /* Maximum number of addresses */
 
 #ifndef LOCKNAME
-#    define LOCKNAME "/etc/sntp.pid"  /* Stores the pid */
+#    define LOCKNAME "/etc/ntp/sntp.pid"  /* Stores the pid */
 #endif
 #ifndef SAVENAME
-#    define SAVENAME "/etc/sntp.state" /* Stores the recovery state */
+#    define SAVENAME "/etc/ntp/sntp.state" /* Stores the recovery state */
 #endif
 
 
--- ./parseutil/dcfd.c.orig	2006-12-04 22:00:05.000000000 +0200
+++ ./parseutil/dcfd.c	2006-12-04 22:02:13.000000000 +0200
@@ -1410,7 +1410,7 @@
 	char **a = argv;
 	int  ac = argc;
 	char *file = NULL;
-	const char *drift_file = "/etc/dcfd.drift";
+	const char *drift_file = "/etc/ntp/dcfd.drift";
 	int fd;
 	int offset = 15;
 	int offsets = 0;
--- ./libntp/audio.c.orig	2006-12-04 22:04:22.000000000 +0200
+++ ./libntp/audio.c	2006-12-04 22:04:58.000000000 +0200
@@ -53,7 +53,7 @@
 static struct audio_device device; /* audio device ident */
 #endif /* HAVE_SYS_AUDIOIO_H */
 #ifdef PCM_STYLE_SOUND
-# define INIT_FILE "/etc/ntp.audio"
+# define INIT_FILE "/etc/ntp/ntp.audio"
 int agc =	SOUND_MIXER_WRITE_RECLEV; /* or IGAIN or LINE */
 int monitor =	SOUND_MIXER_WRITE_VOLUME; /* or OGAIN */
 int devmask = 0;
--- ./ntpdate/ntptimeset.c.orig	2006-12-04 22:07:37.000000000 +0200
+++ ./ntpdate/ntptimeset.c	2006-12-04 22:08:06.000000000 +0200
@@ -270,7 +270,7 @@
 
 #ifndef KEYFILE
 # ifndef SYS_WINNT
-#define KEYFILE 	"/etc/ntp.keys"
+#define KEYFILE 	"/etc/ntp/ntp.keys"
 # else
 #define KEYFILE 	"%windir%\\ntp.keys"
 # endif /* SYS_WINNT */
--- ./ntpdate/ntptime_config.c.orig	2006-12-04 22:06:24.000000000 +0200
+++ ./ntpdate/ntptime_config.c	2006-12-04 22:07:14.000000000 +0200
@@ -37,7 +37,7 @@
  */
 #ifndef CONFIG_FILE
 # ifndef SYS_WINNT
-#  define	CONFIG_FILE "/etc/ntp.conf"
+#  define	CONFIG_FILE "/etc//ntp/ntp.conf"
 # else /* SYS_WINNT */
 #  define	CONFIG_FILE 	"%windir%\\ntp.conf"
 #  define	ALT_CONFIG_FILE "%windir%\\ntp.ini"
--- ./ntpdate/ntpdate.c.orig	2006-12-04 22:05:22.000000000 +0200
+++ ./ntpdate/ntpdate.c	2006-12-04 22:06:07.000000000 +0200
@@ -196,7 +196,7 @@
 
 #ifndef KEYFILE
 # ifndef SYS_WINNT
-#define KEYFILE 	"/etc/ntp.keys"
+#define KEYFILE 	"/etc/ntp/ntp.keys"
 # else
 #define KEYFILE 	"%windir%\\ntp.keys"
 # endif /* SYS_WINNT */
--- ./include/ntp_config.h.orig	2006-12-04 22:02:53.000000000 +0200
+++ ./include/ntp_config.h	2006-12-04 22:03:30.000000000 +0200
@@ -3,7 +3,7 @@
  */
 #ifndef CONFIG_FILE
 # ifndef SYS_WINNT
-#  define	CONFIG_FILE "/etc/ntp.conf"
+#  define	CONFIG_FILE "/etc/ntp/ntp.conf"
 # else /* SYS_WINNT */
 #  define	CONFIG_FILE	"%windir%\\system32\\drivers\\etc\\ntp.conf"
 #  define	ALT_CONFIG_FILE "%windir%\\ntp.conf"
